.devcontainer/Dockerfile

```
@@ -40,13 +40,17 @@ FROM mcr.microsoft.com/devcontainers/go:1.24-bookworm
 
 ARG USERNAME
 
-RUN apt-get update \
-    && apt-get install -y \
+RUN set -eux; \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)